src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.hpp

```diff
@@ -0,0 +1,70 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <memory>
+#include <string>
+#include <unordered_map>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "Domain/Creators/DomainCreator.hpp"
+#include "Domain/Domain.hpp"
+#include "Domain/OptionTags.hpp"
+#include "Domain/Structure/ExcisionSphere.hpp"
+#include "Options/Options.hpp"
+
+namespace CurvedScalarWave::Worldtube {
+/*!
+ * \brief Option tags for the worldtube
+ */
+namespace OptionTags {
+/*!
+ * \brief Options for the worldtube
+ */
+struct Worldtube {
+  static constexpr Options::String help = {"Options for the Worldtube"};
+};
+/*!
+ * \brief Name of the excision sphere desginated to act as a worldtube
+ */
+struct ExcisionSphere {
+  using type = std::string;
+  static constexpr Options::String help{
+      "The name of the excision sphere as returned by the domain."};
+  using group = Worldtube;
+};
+
+}  // namespace OptionTags
+
+/*!
+ * \brief Tags related to the worldtube
+ */
+namespace Tags {
+/*!
+ * \brief The excision sphere corresponding to the worldtube
+ */
+template <size_t Dim>
+struct ExcisionSphere : db::SimpleTag {
```

Comment:
Do you prefer this name over `Worldtube`? Up to you

Reply:
yes, I think so. The worldtube has a lot more properties than the `ExcisionSphere` which are mostly in the databox of the Singleton which I will add.